pkgs/tools/filesystems/btrfs-progs/default.nix

```diff
@@ -9,17 +9,20 @@
 
 stdenv.mkDerivation rec {
   pname = "btrfs-progs";
-  version = "6.6.2";
+  version = "6.6.3";
 
   src = fetchurl {
     url = "mirror://kernel/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v${version}.tar.xz";
-    hash = "sha256-tCZzbclOKwS2mD6ZjE3EJ9wWbWbd6iRsiQ7ZwqRQBE8=";
+    hash = "sha256-9BzlP2Zz/1Ue5KP+fclgHloN3mttCRd9H6ticYq8bZo=";
   };
 
   nativeBuildInputs = [
     pkg-config
   ] ++ [
-    sphinx
+    (python3.withPackages (ps: with ps; [
```

Comment:
this doesn't work for cross compilation (`nix build '.#pkgsCross.aarch64-multiplatform.btrfs-progs'`):
```
sphinx-build: cannot execute binary file: Exec format error
```

using `buildPackages.python3` in place of `python3` here fixes it.

Reply:
Sounds reasonable. Do you want to send a PR or should I?

Reply:
Proposed the PR as https://github.com/NixOS/nixpkgs/pull/275078